i18n/locale-info.php

```diff
@@ -483,6 +483,16 @@
 			),
 		),
 	),
+	'PY' => array(
+		'currency_code'  => 'PYG', // G uppercase striped
+		'currency_pos'   => 'left',
+		'thousand_sep'   => '.',
+		'decimal_sep'    => ',',
+		'num_decimals'   => 2,
+		'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
+		'tax_rates'      => array(),
```

Comment:
While we are defining the locale info, let's also add the VAT rates http://world.salestaxhandbook.com/py-paraguay